cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -195,6 +214,40 @@ To configure a custom SAML authentication method:
 1. Download metadata required by your IdP. Click **Download**. Open the file and make a note of the following values:<ul><li><b>Entity ID</b>: The <code>entityID</code> attribute of the <code>&lt;EntityDescriptor&gt;</code> tag.</li><li><b>Login URL</b>: The <code>location</code> attribute of the <code>&lt;AssertionConsumerService&gt;</code> tag.</li></ul>
 1. In the browser where you are logged in to your IdP, update the authentication configuration to use the Entity ID and Login URL from the metadata file.
 1. Configure the SAML assertions that your IdP sends to {{ site.data.products.db }}. Your IdP must send an assertion with a `name` field and a second assertion with an `email` field, each mapped to the relevant fields in your IdP. Refer to the documentation for your IdP.
+1. To configure SCIM provisioning, refer to [(Optional) Configure SCIM autoprovisioning on Okta](#configure-scim-provisioning).
+
+<a id="configure-scim-provisioning"></a>
+
+#### (Optional) Configure SCIM autoprovisioning on Okta
+
+The exact steps and requirements for enabling SCIM autoprovisioning depend upon your IdP. To add support for [SCIM autoprovisioning](#scim-autoprovisioning) on Okta:
+
+1. Log in to Okta Admin Dashboard as an admin user.
+1. Click **Applications** and edit the [SAML](#saml) application for your {{ site.data.products.db }} organization.
+1. Edit the SAML application, then click **Provisioning**:
+
+      {{site.data.alerts.callout_danger}}
+      Cockroach Labs does not recommend enabling both [SSO autoprovisioning](#sso-autoprovisioning) and SCIM autoprovisioning for the same authentication method.
+      {{site.data.alerts.end}}
+
+1. The **To App** tab controls autoprovisioning from Okta to {{ site.data.products.db }}. Ensure that **Create Users** and **Deactivate Users** are selected, and make any other desired changes.
+1. The **To Okta** tab controls autoprovisioning from {{ site.data.products.db }} to Okta. For example, if you create a new user in {{ site.data.products.db }}, you can automatically provision an Okta user from that {{ site.data.products.db }} user. Make any desired changes.
+1. The **Integration** tab controls the SCIM app's authentication to the {{ site.data.products.db }} API.
+
+    1. Set **SCIM connector base URL** to the base URI for the {{ site.data.products.db }} API plus the `/api/scim/v2` endpoint. For example, `https://cockroachlabs.cloud/api/scim/v2`.
+    1. Set the API authentication token to the API token for a {{ site.data.products.db }} [service account](console-access-management.html#service-accounts) with the `ADMIN` role.
+    1. Click **Test Connector Configuration**.
+    1. Click **Save**.
+
+1. After configuring SCIM autoprovisioning, click **Assignments** to autoprovision a {{ site.data.products.db }} account for a SCIM user or members of a SCIM group to the SAML application:
```

Comment:
I'm not sure about the user of SCIM user and SCIM group here. It feels like it's referring to an IDP's users and groups.